pomanders.md

```diff
@@ -87,6 +87,10 @@ pomanders:
       - 'This is a good panic button when you agro a second mob or hit a luring
       trap'
       - 'Be aware that imps still do significant damage'
+        - 'Imp: 60% base autoattack damage'
+        - 'Chicken: 30% base autoattack damage'
+        - 'Frog: negligible (<1%) damage'
```

Comment:
Multi-level lists aren't supported here. I just pushed a change that will allow it, but it needs to be formatted as note/subnotes:

```suggestion
      - note: 'Be aware that imps still do significant damage'
        subnotes:
        - 'Imp: 60% base autoattack damage'
        - 'Chicken: 30% base autoattack damage'
        - 'Frog: negligible (<1%) damage'
```